templates/yao/meta.yaml

```diff
@@ -0,0 +1,39 @@
+name: YAO
+description: YAO is an open-source application engine,
+instructions: Use xiang@iqka.com:A123456p+ to login at /xiang/login/admin . Use amd64 instead of arm64 if you use a non ARM based system.
```

Comment:
is there a way to change the default credentials using env vars?

Reply:
this is hardcoded sadly and there is no way to change it as of now.